org-brain.el

```diff
@@ -62,24 +64,116 @@ This will be used by `org-brain-new-child'."
   :group 'org-brain
   :type '(string))
 
+;;; Utils
+(defun flatten (obj)
+  (do* ((result (list obj))
+        (node result))
+       ((null node) (delete nil result))
+    (cond ((consp (car node))
+           (when (cdar node) (push (cdar node) (cdr node)))
+           (setf (car node) (caar node)))
+          (t (setf node (cdr node))))))
+
+(defun trim (string)
```

Comment:
Thanks. I totally missed that. Was going fast : ) trim was from some of my Common Lisp code.